core/impl/src/main/java/pt/webdetails/cdf/dd/model/inst/writer/cdfrunjs/legacy/CdfRunJsThingWriterFactory.java

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2002 - 2017 Webdetails, a Hitachi Vantara company. All rights reserved.
+ * Copyright 2002 - 2018 Webdetails, a Hitachi Vantara company. All rights reserved.
```

Comment:
Don't think we should be refactoring the legacy code. The new `CdfRunJsThingWriterBaseFactory` base class serves only to unify the legacy with the amd code, when the idea, IMO, would be to keep the two as much independent as possible, so that, one day, we can simply delete the "legacy" folder. This way, we'll be left with a base class which will serve a single derived class. Was Solr support added to legacy dashboards? Or, is there no way to only expose it to AMD dashboards?